### PR TITLE
Simple is better than complex!

### PR DESCRIPTION
**Pull request by Niedzwiedzw**

### CODE REVIEW

File: rock_paper_scissors_by_p_stoyanov.py [hunk 1]
```diff
@@ -14,8 +14,9 @@
 COMPUTER_RANDOM_NUMBER = random.randint(1, 5)
 
 command = input(colored("Choose [r]ock, [p]aper, [s]cissors [l]izard or Sp[o]ck : ", 'red'))
-while command not in "r" and command not in "p" and command not in "s" and command not in "l" and command not in "o":
+while command not in ["r", "p", "s", "l", "o"]:
```
Comment (Niedzwiedzw) [inline rock_paper_scissors_by_p_stoyanov.py:17]: this is the same, but shorter and easier to read, do you agree?

Comment (PeterStoyanov83) [reply rock_paper_scissors_by_p_stoyanov.py:17]: thanks. This makes sense  now that I went through lists :

File: rock_paper_scissors_by_p_stoyanov.py [hunk 2]
```diff
@@ -2,115 +2,115 @@
...
+ROCK = "Rock"
+PAPER = "Paper"
+SCISSORS = "Scissors"
+LIZARD = "Lizard"
+SPOCK = "Spock"
+COMPUTER_MOVE = ""
+PLAYER_MOVE = ""
+COMPUTER_POINTS = 0
+PLAYER_POINTS = 0
+COMPUTER_RANDOM_NUMBER = random.randint(1, 5)
```
Comment (PeterStoyanov83) [inline rock_paper_scissors_by_p_stoyanov.py:15]: i haven't yet touched the topic of Global Variables. I know they exist but i don't know how to use them yet.

Comment (Niedzwiedzw) [reply rock_paper_scissors_by_p_stoyanov.py:15]: I mean its just a convention

**Review by PeterStoyanov83 — COMMENTED** [review]
thank you

**Pull request merged**